### DISCUSSION TITLE
Game freezes at specific points

### DISCUSSION THREAD

**Discussion by hwabis** [ppy/osu#16315] [Q&A] [answered]
When I select a map from this set https://osu.ppy.sh/beatmapsets/115107, my osu freezes immediately, so I have to force close the game. Sometimes my screen flickers rapidly, but the most recent one (in the logs) was a simple freeze. I've tried a few other sets and they've all worked. 

Using Windows 11, so maybe that's the problem?

Also, this only happens in fullscreen, not borderless nor windowed.

[updater.log](https://github.com/ppy/osu/files/7804488/updater.log)
[database.log](https://github.com/ppy/osu/files/7804489/database.log)
[network.log](https://github.com/ppy/osu/files/7804490/network.log)
[performance.log](https://github.com/ppy/osu/files/7804491/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7804492/runtime.log)

Edit: also freezing consistently for this set https://osu.ppy.sh/beatmapsets/5419

**Comment by peppy** [comment, marked as answer]
From the logs, I would guess that your hardware video decoding is falling over. You can try turning this off in the settings (search for "video" and disable hardware acceleration) to test.

If it turns out to be the issue, I'd recommend updating your graphics card drivers, as that may fix the underlying issue.

**Reply by hwabis** [reply]
I disabled hardware acceleration, but I'm still having the same problem. Here are more logs after disabling, if it helps. (I just find it strange that osu has only froze for this mapset.)

[database.log](https://github.com/ppy/osu/files/7805154/database.log)
[network.log](https://github.com/ppy/osu/files/7805155/network.log)
[performance.log](https://github.com/ppy/osu/files/7805156/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7805157/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7805158/updater.log)

**Reply by peppy** [reply]
Can you confirm that it also happens if you disable video completely?

**Reply by hwabis** [reply]
Yes, I disabled storyboard / video in settings, but it still froze. 

Also, sometimes it freezes after pressing "play --> solo" in the title screen (logs below). Still, everything is fine when I'm not fullscreen.

[updater.log](https://github.com/ppy/osu/files/7805172/updater.log)
[database.log](https://github.com/ppy/osu/files/7805173/database.log)
[network.log](https://github.com/ppy/osu/files/7805174/network.log)
[performance.log](https://github.com/ppy/osu/files/7805175/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7805176/runtime.log)

**Reply by peppy** [reply]
Sounds like it may be related to an external overlay (discord / nvidia experience / overwolf / etc.) in that case, as there's nothing showing up in the logs. Please try disabling any overlays you have running and see if it helps with the crash.

**Reply by hwabis** [reply]
I had Discord on, so I closed it (and as many other applications I could), but it's still freezing.

Sometimes after the freeze, the whole game keeps running except my screen is stuck on one frame. I can tell because of the audio effects from interacting with the game (like running the mouse over the beatmap select panels or the song that's playing).

**Reply by peppy** [reply]
Please update your graphics card drivers. It might resolve the issue. You can also check windows event viewer to hopefully find more details (it definitely sounds like a driver level failure).

**Reply by hwabis** [reply]
Thanks, I think it worked! I will remember to update my drivers in the future.
